api/app/signals/apps/signals/serializers.py

```diff
@@ -30,7 +30,17 @@
 log = logging.getLogger(__name__)
 
 
-class LocationModelSerializer(serializers.ModelSerializer):
+class NearAmsterdamValidatorMixin:
+    def validate_geometrie(self, value):
+        fail_msg = 'Location coordinates not anywhere near Amsterdam. (in WGS84)'
+
+        if (not 1 < value.coords[0] < 7) or (not 50 < value.coords[1] < 55):
+            raise ValidationError(fail_msg)
```

Comment:
For DRF validation errors we should raise a `serializers.ValidationError` exception. See http://www.django-rest-framework.org/api-guide/validators/#writing-custom-validators

Reply:
fixed as suggested